privacy_policy.html

```diff
@@ -0,0 +1,267 @@
+    <meta name="description" content="Pocket Science Lab">
+    <meta name="keywords" content="PSLab, Pocket Science Lab, Android, Python Science Lab, ExpEYES, FOSSASIA">
+    <meta name="author" content="">
+    <title>Pocket Science Lab</title>
+    <!-- Bootstrap Core CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <!-- Theme CSS -->
+    <link href="css/clean-blog.min.css" rel="stylesheet">
+    <!-- Custom Fonts -->
+    <script src="https://use.fontawesome.com/f219d2cd7c.js"></script>
+    <link href='https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic' rel='stylesheet'
+          type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800'
+          rel='stylesheet' type='text/css'>
+    <link href="css/custom.css" rel="stylesheet">
+    <link rel="icon" type="image/png" href="img/favicon.png">
+</head>
+
+<body>
+<!-- Navigation -->
+<nav class="navbar navbar-default navbar-custom navbar-static-top" id="sticky-navbar">
+    <div class="container-fluid">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header navbar-static-top page-scroll">
+            <button type="button" class="navbar-toggle" data-toggle="collapse"
+                    data-target="#bs-example-navbar-collapse-1">
+                <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
+            </button>
+        </div>
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <a href="http://fossasia.org" id="light-logo" class="link-logo"><img src="img/fossasia-logo-light.png"
+                                                                                 alt="fossasia logo" class="logo-navbar"
+                                                                                 style="cursor: pointer;"></a>
+            <a href="http://fossasia.org" id="dark-logo" class="link-logo"><img src="img/fossasia-logo.png"
+                                                                                alt="fossasia logo" class="logo-navbar"
+                                                                                style="cursor: pointer;"></a>
+            <ul class="nav navbar-nav navbar-right">
+                <li id="fossasia-link">
+                    <a href="http://fossasia.org" target="_blank">FOSSASIA</a>
+                </li>
+                <li>
+                    <a href="index.html" title="Home">Home</a>
+                </li>
+                <li>
+                    <a href="#features" title="Features">Features</a>
```

Comment:
Hi @viveksb007 ! I guess this link needs to be changed right? As this points to a div. Does it work?